plugins/woocommerce/tests/php/src/Gateways/PayPal/RequestTests.php

```diff
@@ -298,7 +298,7 @@ public function provide_test_get_paypal_order_details(): array {
 	 * @dataProvider provide_test_get_paypal_order_purchase_unit_amount
 	 */
 	public function test_get_paypal_order_purchase_unit_amount( int $cart_tax, int $shipping_tax, int $discount_total, int $total, array $expected ): void {
-		$order = \WC_Helper_Order::create_order();
+		$order = new \WC_Order();
```

Comment:
@wjrosa, this test is failing due to this change. With `\WC_Helper_Order::create_order()` this tests pass locally for me.

Reply:
But it still fails